src/evergreen/build.py

```diff
@@ -9,7 +9,7 @@
 
 if TYPE_CHECKING:
     from evergreen.api import EvergreenApi
-    from evergreen.task import Task
+    from evergreen.task import Task  # noqa: F401
```

Comment:
flake8 isn't smart enough to know this is being used when embedded in other types (e.g. `List["Task"]`), but IDEs understand what is going on.